test/test_room_db_repository.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from application.entities.room_entity import RoomEntity
+from room.models import RoomModel
+from room.repository import RoomDBRepository
+
+
+@pytest.mark.django_db
+def test_get_all_rooms(populate_penthouse_and_basement_rooms_to_db):
```

Comment:
We certainly want to test that the repository properly works with the database engine.